examples/render_all_grammars/render_all_grammars.py

```diff
@@ -16,6 +16,9 @@ def main(path=None, debug=False, reportfilename=None):
     print("render_all_grammars.py - example program")
     matches = []
     for root, dirnames, filenames in os.walk(path):
+        if 'textx_textx' in root:
+            # skip semantically invalid grammars
```

Comment:
Can you give me a hint for the rationale of this change?

Reply:
That change leave out grammar from the tests of the implementation of textX in textX. Those grammars are there just to check that syntactically correct textX grammars can be parsed and analyzed programmatically but are not fully semantically correct. To be precise link rule references don't have valid target rules always. I thought that the easiest fix is to skip those grammars.